setup.cfg

```diff
@@ -115,6 +115,7 @@ install_requires =
     # Logging is broken with itsdangerous > 2
     itsdangerous>=1.1.0, <2.0
     jinja2>=2.10.1,<4
+    jsonpath_ng>=1.5.3
```

Comment:
If there's another lib we prefer for JSONPath expression parsing let me know and I'll try to rework this to use it. This was the best one I could find.

Reply:
We already used it in the past and **just** removed it few weeks ago as it was not needed any more ;). So I think it's good to go.

Reply:
However, rather than in setup.cfg it should be added in setup.py for amazon provider.https://github.com/apache/airflow/blob/df0746e133ca0f54adb93257c119dd550846bb89/setup.py#L182

Reply:
Fixed up, thank you for the feedback